paramid_string.go

```diff
@@ -0,0 +1,141 @@
+        _ = x[ColorFormatParam-36438016]
+        _ = x[ConnectorParam-11173888]
+        _ = x[DriverIndexParam-0]
+        _ = x[ElapsedPgParam-56606720]
+        _ = x[EncoderPitchParam-9699328]
+        _ = x[ErrorHandlingParam-802816]
+        _ = x[ErrorLogParam-1327104]
+        _ = x[ForceTrigParam-819200]
+        _ = x[ImageSizeXParam-8568832]
+        _ = x[ImageSizeYParam-8585216]
+        _ = x[LinePitchParam-9748480]
+        _ = x[LineRateModeParam-21757952]
+        _ = x[MetadataContentParam-177750016]
+        _ = x[MetadataGPPCInputLineParam-183681024]
+        _ = x[MetadataGPPCLocationParam-183713792]
+        _ = x[MetadataGPPCResetLineParam-183730176]
+        _ = x[MetadataInsertionParam-177635328]
+        _ = x[MetadataLocationParam-181125120]
+        _ = x[MinBufferPitchParam-54640640]
+        _ = x[OutputConfigParam-28508160]
+        _ = x[OutputStateParam-28540928]
+        _ = x[PeriodUsParam-21774336]
+        _ = x[SerialNumberParam-81920]
+        _ = x[SeqLengthFrParam-55820288]
+        _ = x[SeqLengthPgParam-55836672]
+        _ = x[SeqLengthLnParam-55721984]
+        _ = x[SignalEnableParam-393216]
+        _ = x[SurfaceAddrParam-458752]
+        _ = x[SurfaceCountParam-1343488]
+        _ = x[SurfaceIndexParam-278528]
+        _ = x[SurfacePitchParam-475136]
+        _ = x[SurfaceSizeParam-442368]
+        _ = x[SurfaceStateParam-507904]
+        _ = x[AnySignal-0]
+        _ = x[StartAcquisitionSignal-11]
+        _ = x[EndAcquisitionSignal-10]
+        _ = x[AcquisitionFailureSignal-7]
+        _ = x[ClusterUnavailableSignal-8]
+        _ = x[EndChannelActivitySignal-12]
+        _ = x[FrameTriggerViolationSignal-4]
+        _ = x[SurfaceProcessingSignal-1]
+        _ = x[SurfaceFilledSignal-2]
+        _ = x[StartExposureSignal-5]
+        _ = x[EndExposureSignal-6]
+        _ = x[UnrecoverableOverrunSignal-3]
+        _ = x[ReleaseSignal-9]
+}
+
+const _ParamID_name = "DriverIndexParamSurfaceProcessingSignalSurfaceFilledSignalUnrecoverableOverrunSignalFrameTriggerViolationSignalStartExposureSignalEndExposureSignalAcquisitionFailureSignalClusterUnavailableSignalReleaseSignalEndAcquisitionSignalStartAcquisitionSignalEndChannelActivitySignalBoardPCIPositionParamBoardNameParamBoardIdentifierParamBoardNameChangeParamBoardSerialNumberParamBoardTypeParamCamFileParamClusterParamChannelStateParamSurfaceIndexParamSignalEnableParamSurfaceSizeParamSurfaceAddrParamSurfacePitchParamSurfaceStateParamErrorHandlingParamForceTrigParamBoardCountParamErrorLogParamSurfaceCountParamImageSizeXParamImageSizeYParamEncoderPitchParamLinePitchParamConnectorParamLineRateModeParamPeriodUsParamOutputConfigParamOutputStateParamBoardIndexParamColorFormatParamBufferSizeParamMinBufferPitchParamBufferPitchParamAcquisitionModeParamSeqLengthLnParamSeqLengthFrParamSeqLengthPgParamElapsedPgParamMetadataInsertionParamMetadataContentParamMetadataLocationParamMetadataGPPCInputLineParamMetadataGPPCLocationParamMetadataGPPCResetLineParam"
```

Comment:
Where did you get that one from?

Reply:
https://github.com/northvolt/go-multicam/pull/17/files#diff-0778a82931391185985d0798b65f5d73dcdc4b738e6b3055e50654ca9bc88926R48

https://pkg.go.dev/golang.org/x/tools/cmd/stringer